### PR TITLE
added-game-files

### PR DESCRIPTION
**Pull request by murtazshaikh**

### CODE REVIEW

**Review by syedabdullahali — APPROVED** [review]

File: script.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+const player2El = document.querySelector(".player--2");
+const score1El = document.querySelector("#score--1");
+const score2El = document.getElementById("score--2");
+const current1El = document.getElementById('current--1');
+const current2El = document.getElementById('current--2');
+
+const diceEl = document.querySelector('.dice');
+const btnNew = document.querySelector('.btn--new');
+const btnRoll = document.querySelector('.btn--roll');
+const btnHold = document.querySelector('.btn--hold');
```
Comment (Abbas4tech) [inline script.js:13]: Can we use `querySelectorAll` for this ?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+let scores, currentScore, activePlayer, playing;
+
+const init = function(){
```
Comment (Abbas4tech) [inline script.js]: Use arrow functions

File: script.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+  diceEl.classList.add('hidden');
+  player1El.classList.remove('player--winner');
+  player2El.classList.remove('player--winner');
+  player1El.classList.add('player--active');
+  player2El.classList.remove('player--active');
+};
+init();
+
+const switchPlayer = function(){
```
Comment (Abbas4tech) [inline script.js]: Use arrow functions

File: script.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    if(dice !== 1){
+      currentScore += dice;
+      document.getElementById(`current--${activePlayer}`).textContent = currentScore;
+    }else{
+      switchPlayer();
+    }
+  }
+});
+
+btnHold.addEventListener('click', function(){
```
Comment (Abbas4tech) [inline script.js]: Use clickhandler for these and write code inside that function

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Please fix above issues

**Review by ManSOorcode — APPROVED** [review]

File: script.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
+"use strict";
```
Comment (anask601) [inline script.js:1]: what is use of use strict ?

Comment (murtazshaikh) [reply script.js:1]: It defines that JavaScript should be executed in "strict mode".
For example, with strict mode we cannot use undeclared variables.

File: script.js (hunk 1 above)
Comment (murtazshaikh) [reply script.js:13]: Yes ! But now just to keep it simple I have selected separately.

File: script.js (hunk 2 above)
Comment (murtazshaikh) [reply script.js]: Done

File: script.js (hunk 3 above)
Comment (murtazshaikh) [reply script.js]: Done

File: script.js (hunk 4 above)
Comment (murtazshaikh) [reply script.js]: Changed

**Review by samar4tech — APPROVED** [review]

File: style.css [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+  margin: 0;
+  padding: 0;
+
+  font-size: 62.5%;
+
+  box-sizing: border-box;
+}
+
+body{
+  height: 100vh;
```
Comment (zaidansari42) [inline style.css:13]: What would you suggest, is it better to use a fixed height here, or a min-height ?

File: style.css [hunk 7]
```diff
@@ -0,0 +1,233 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+
+*{
+  margin: 0;
+  padding: 0;
+
+  font-size: 62.5%;
```
Comment (zaidansari42) [inline style.css:7]: Why did you prefer using a relative font-size rather than absolute size ?

**Review by zaidansari42 — APPROVED** [review]

File: style.css (hunk 6 above)
Comment (murtazshaikh) [reply style.css:13]: I personally prefer this to make my content fixed to the 1 page only

File: style.css (hunk 7 above)
Comment (murtazshaikh) [reply style.css:7]: To make it responsive

**Pull request merged**